src/style.css

```diff
@@ -1 +1,135 @@
-/* add styles here */
+* {
```

Comment:
Using the universal selector '*' is not recommended as it impacts performance. Consider applying styles to specific elements or classes instead.